Sources/Fuzzilli/FuzzIL/JsOperations.swift

```diff
@@ -2064,6 +2064,22 @@ final class LoopContinue: JsOperation {
     }
 }
 
+final class BreakNested: JsOperation {
```

Comment:
Is my understanding correct that these can _only_ be used inside loops? Or are they also usable outside of loops in certain circumstances?

If they must always be in a loop: maybe we should also call these `LoopBreakNested` or something like that to stay consistent with `LoopBreak`

If they can also appear elsewhere: should we drop the `.loop` context requirement?

Reply:
Yes, these can only be used in loops. This is an inappropriate naming issue, and I will later rename it to `LoopBreakNested` and `LoopContinueNested`.